openfecwebapp/filters.py

```diff
@@ -30,6 +30,11 @@ def date_filter(value, fmt='%m/%d/%Y'):
         return None
     return ensure_date(value).strftime(fmt)
 
+@app.template_filter('ao_document_date')
+def ao_document_date(value):
+    date = date_filter(value)
+    return 'Not dated' if date == '01/01/1900' else date
```

Comment:
Not sure what you mean. The raw data coming back from api goes through date_filter function in line 35, that formats it.